src/native/libs/System.IO.Compression.Native/extra_libs.cmake

```diff
@@ -19,4 +19,12 @@ macro(append_extra_compression_libs NativeLibsExtra)
     list(APPEND ZLIB_LIBRARIES $<IF:$<BOOL:${CLR_CMAKE_USE_SYSTEM_ZLIB}>,z,zlib>)
   endif ()
   list(APPEND ${NativeLibsExtra} ${ZLIB_LIBRARIES})
+
+  if (CLR_CMAKE_USE_SYSTEM_BROTLI)
+    find_library(BROTLIDEC brotlidec REQUIRED)
+    find_library(BROTLIENC brotlienc REQUIRED)
```

Comment:
Does the updated change look okay?

Reply:
Wait, this doesn't work at all :(

Reply:
Ah, because src/native/corehost/apphost/static/CMakeLists.txt only includes `NATIVE_LIBS`

https://github.com/dotnet/runtime/blob/d7e85d93c12aaa43937f8ef5f920c36b5afc756d/src/native/corehost/apphost/static/CMakeLists.txt#L183-L185

Should it include `BROTLI_LIBRARIES` under a `CLR_CMAKE_USE_SYSTEM_BROTL` conditional?

Reply:
The `BROTLI_LIBRARIES` values should be propagated up to the static apphost due to

https://github.com/dotnet/runtime/blob/d7e85d93c12aaa43937f8ef5f920c36b5afc756d/src/native/libs/System.IO.Compression.Native/CMakeLists.txt#L99

and

https://github.com/dotnet/runtime/blob/d7e85d93c12aaa43937f8ef5f920c36b5afc756d/src/native/libs/System.IO.Compression.Native/CMakeLists.txt#L152

You can change it back to how it was before to unblock and I can investigate later.